bot/potentialScammer/getTargetChannelId.js

```diff
@@ -0,0 +1,19 @@
+const getChannelList = (guild) => {
+  //get all channel of a guild
+  const channelList = guild.channels.cache
+  return channelList
+}
+
+const getTargetChannelId = async (guild) => {
+  //get the target channel id
+  let targetChannelId
+  const channelList = await getChannelList(guild)
+  channelList.forEach(channel => {
```

Comment:
Extract this filter logic into another file which doesn't rely on discord, add unit test.